### PR TITLE
Remove govuk_frontend_toolkit dependency

### PR DESCRIPTION
**Pull request by andysellick**
This PR removes the `frontend` Sass dependency on `govuk_frontend_toolkit`. This has involved significant HTML and CSS restructuring to some parts of the application, but hopefully minimal visual changes (if any).

(was doing this as part of https://github.com/alphagov/frontend/pull/1927 but it's turned into too big a bit of work, so splitting it out to do first)

I want to do more changes to frontend, because there's more to be rebuilt using `govuk-frontend`, but this PR seems big enough for now so I'll do those later (some have been done as part of 1927 already). The main focus of this PR is updating mixins and rebuilding the home page.

## Homepage desktop before

image

## Homepage desktop after

The black column in the header is slightly wider and in a slightly different position. This is due to the fact that this element is absolutely positioned and the containers have changed slightly. I don't think this is worth the effort required to fix it (it's a few pixels)

image

## Homepage mobile before

image

## Homepage mobile after

Should be the same, but please check.

image

### CODE REVIEW

File: app/assets/stylesheets/views/_travel-advice.scss [hunk 1]
```diff
@@ -238,64 +230,10 @@
...
-      float: none;
-      text-align: left;
-      width: auto;
-
-      @include media-down(mobile) {
-        padding-left: 1em;
-      }
-    }
+  .page-navigation {
+    border-bottom: 1px solid #bbb;
```
Comment (alex-ju) [inline app/assets/stylesheets/views/_travel-advice.scss]: Is there a reason for not using a standard colour for this (e.g. `$govuk-border-colour`)?

Comment (andysellick) [reply app/assets/stylesheets/views/_travel-advice.scss]: Updated.

**Comment by andysellick** [comment]
@alex-ju was all ready to merge and then checked it in IE8. We've been having a conversation this week about support for older browsers (some 40,000 people are still on <= IE8) and govuk-frontend's media queries no longer support IE8 and below, meaning that this change will take those users from this:

Screen Shot 2019-09-05 at 11 19 56

to this:

Screen Shot 2019-09-05 at 11 20 10

(media queries are ignored, so they get the mobile version, but without the max-width on the container, so it all collapses left)

Any thoughts?

**Comment by alex-ju** [comment]
@andysellick looks quite bad. I would add `max-width` and `width: auto` (and other specific CSS if needed) in the IE8 stylesheet.

**Comment by andysellick** [comment]
@alex-ju I've added some IE specific styles to make the homepage look more like it should. I've also finally figured out why compiling all this stuff takes so long - see the last commit in this PR (we could adopt this on other repos).

**Comment by alex-ju** [comment]
Good find @andysellick! 👏 Good to be merged when e2e tests are passing.

**Comment by andysellick** [comment]
@alex-ju have also realised that all the `$ie` variables in those stylesheets are now unnecessary, so have removed.

**Pull request merged**